lib/license_finder/package_managers/conda.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'pry'
+
+module LicenseFinder
+  class Conda < PackageManager
+    attr_reader :conda_bash_setup_script
+
+    def initialize(options = {})
+      @conda_bash_setup_script = options[:conda_bash_setup_script] || Pathname('/root/miniconda3/etc/profile.d/conda.sh')
```

Comment:
Asking more for curiosity. Is hardcoding this path the right thing to do? Does this script normally sit here? I do like how you made this customizable

Reply:
Thanks for asking. This is a reasonable default since it matches up to where it's installed in the Dockerfile, but there's no predictable "normal" place (other than `/etc/profile.d`, but many [all of our] developers install to `~/miniconda3`).  OTOH, I think there's a way to simplify this handling and will make an attempt to do so.

Reply:
Unfortunately, the best I could do was make the default be based on $HOME instead of hardcoding `/root/`.